R/convertMT.R

```diff
@@ -0,0 +1,115 @@
+#' @title Convert migration times to effective mobility
+#'
+#' @description
+#'
+#' `convertMT` uses a list of known EOF markers with to convert migration times 
+#' (MTs) to effective mobilities (µeff). By using the information of the EOF 
+#' markers (which is their MT and µeff), it is possible to transform the MT-
+#' scale (for consistency called rtime) into the µeff scale. This is used in 
+#' CE-MS to overcome MT shifts originating from EOF fluctuations in different 
+#' runs.
+#'
+#' @param x `numeric` vector with migration times in minutes.
+#'
+#' @param y `data.frame`data.frame containing minimum two columns, where one
+#'     holds the determined migration time in minutes (here referred to as 
+#'     `rtime`) of the EOF marker in the same run in which the migration time is 
+#'     going to be transformed and the other column the respective mobility of 
+#'     the EOF markers. Each row hold the values for one EOF marker. The minimum 
+#'     of required markers is one. One or two entries are required for the 
+#'     transformation and depending on the number of entries the transformation 
+#'     will be performed either on one or both markers.
+#'
+#' @param FUN `function` function defining how the conversion is performed,
+#'     either a single EOF marker is used (convertSingle) or multiple markers 
+#'     (convertMultiple)
+#'
+#' @return `numeric` vector of same length as x with effective mobility values. 
+#'
+#' @author Liesa Salzer
+#'
+#' @export
+#'
+#' @examples
+#' rtime <- c(10,20,30,40,50,60,70,80,90,100)
+#' marker <- data.frame(markerID = c("marker1", "marker2"),
+#'   rtime = c(20,80),
+#'   mobility = c(0, 2000))
+#' convertMT(rtime, marker)
+
+convertMT <- function(x, y, ...) {
```

Comment:
I agree that it is easier to split the `data.frame` into 2 individual vectors, but if we have 2 markers `rtime` and `mobility` will contain 2 entries and it must be clear that the first entry of both vectors belongs to the same marker. 
That Is the reason why I thought a `data.frame` might be better because here one row corresponds to one marker

Reply:
I agree that it is easier to split the `data.frame` into 2 individual vectors, but if we have 2 markers `rtime` and `mobility` will contain 2 entries and it must be clear that the first entry of both vectors belongs to the same marker. 
That Is the reason why I thought a `data.frame` might be better because here one row corresponds to one marker

Reply:
And also the variables `U` and `L` are only needed if a single marker is used. Shouldn't they be defined for that reason only in the function `convertSingle` and not in `convertMtime`?